pydocumentdb/auth.py

```diff
@@ -58,7 +58,9 @@ def __GetAuthorizationTokenUsingMasterKey(verb,
         dict
 
     """
-    key = master_key.decode('base64')
+    # The master_key from Azure UI which the dev/user specifies is base64
```

Comment:
Please change(in the comments):
master_key -> Primary Key or Secondary Key
Azure UI -> Azure portal.